GotExplorer.DAL/Entities/Game.cs

```diff
@@ -0,0 +1,23 @@
+﻿using GotExplorer.DAL.Entities;
+using GotExplorer.DAL.Interfaces;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace GotExplorer.DAL.Entities
+{
+    public class Game : IEntity<int>
+    {
+        public int Id { get; set; }
+
+        public int Score { get; set; }
+
+        public TimeSpan SpentTime { get; set; }
+
+        [ForeignKey("User")]
```

Comment:
I would recommend to use this attribute  on  the navigation property. 
I would recommend to use this approach or every where or never